mailpoet/assets/js/src/wizard/steps/pitch_mss_step/first_part.tsx

```diff
@@ -0,0 +1,81 @@
+import { useHistory, useParams } from 'react-router-dom';
+import { external, Icon } from '@wordpress/icons';
+import { Heading } from '../../../common/typography/heading/heading';
+import { MailPoet } from '../../../mailpoet';
+import { List } from '../../../common/typography/list/list';
+import { Button } from '../../../common';
+import { OwnEmailServiceNote } from './own_email_service_note';
+
+type MSSStepFirstPartPropType = {
+  subscribersCount: number;
+  finishWizard: (redirect_url?: string) => void;
+};
+
+function MSSStepFirstPart({
+  subscribersCount,
+  finishWizard,
+}: MSSStepFirstPartPropType): JSX.Element {
+  const history = useHistory();
+  const { step } = useParams<{ step: string }>();
+
+  const mailpoetAccountUrl = MailPoet.MailPoetComUrlFactory.getPurchasePlanUrl(
```

Comment:
If i'm not mistaken `mailpoetAccountUrl` is not bound to state, so it can get defined outside of the component.

Reply:
Done